### PR TITLE
Get Command for Polaris-Reporting

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1253#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2020.2.x/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@mphammer: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1253#issuecomment-589807699):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by mphammer** [comment]
Example output:
```
eventstore:
  persistence:
    size: 1Gi
    storageClass: standard
global:
  environment: onpremenv
  rootDomain: domain.synopsys.com
onprem-auth-service:
  smtp:
    sender_email: email@synopsys.com
postgres:
  host: <host>
  isExternal: false
  password:<password>
  size: 1Gi
  storageClass: standard
  user: <username>
rp-storage-service:
  report-storage:
    volume:
      size: 1Gi
      storageClass: standard
smtp:
  host: 'smtp.host.net '
  password: <password>
  port: "2525"
  user: <username>
```

File: pkg/util/helm_helpers.go [hunk 1]
```diff
@@ -246,6 +247,20 @@ func DeleteWithHelm3(releaseName, namespace, kubeConfig string) error {
...
+// GetWithHelm3 uses the helm NewUninstall action to delete a resource from the cluster
+func GetWithHelm3(releaseName, namespace, kubeConfig string) (*release.Release, error) {
+	actionConfig, err := CreateHelmActionConfiguration(kubeConfig, "", namespace)
+	if err != nil {
+		return nil, err
+	}
+	client := action.NewGet(actionConfig)
+	release, err := client.Run(releaseName) // lists the releases in the namespace from the actionConfig
+	if err != nil {
+		return nil, fmt.Errorf("failed to Run NewGet: %+v", err)
```
[the remark this answers is not in the document]
Comment (mphammer) [reply pkg/util/helm_helpers.go]: They should see that message from this line of code: https://github.com/blackducksoftware/synopsys-operator/pull/1253/files#diff-6cdca911ec46c519061fe3b0af127c62R265

**Comment by operator-r2d2** [comment]
@mphammer: new pull request created: #1260

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1253#issuecomment-589807699):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**